ambient.d.ts

```diff
@@ -246,18 +246,38 @@ declare module '@mojaloop/central-services-shared' {
       TP_CONSENT_PUT_ERROR: string;
     };
   }
+
+  enum EventTypeEnum {
+    AUTHORIZATION = 'AUTHORIZATION',
+    NOTIFICATION = 'NOTIFICATION',
+    TRANSACTION_REQUEST = 'TRANSACTION_REQUEST',
+  }
+
+  enum EventActionEnum {
+    EVENT = 'EVENT',
+    POST = 'POST',
+    PUT = 'PUT',
+  }
+
   interface Enum {
     Http: HttpEnum;
     EndPoints: EndPointsEnum;
+    Kafka: {
+      Config: {
+        CONSUMER: string
```

Comment:
why the only CONSUMER where there is also PRODUCER,
shouldn't it be handled as an enum? https://github.com/mojaloop/central-services-shared/blob/master/src/enums/kafka.js#L31